site/en/blog/how-fugu-is-my-browser-and-how-fugu-is-the-web/index.md

```diff
@@ -0,0 +1,108 @@
+
+Many developers and executives alike don't realize how capable the modern web has become.
+Double-clicking an image file so it opens in an associated PWA, making modifications, saving the
+changes back to the file, and then copying the image contents over into another app or sharing it to
+an email client, is a flow that wasn't possible on the web until recently. But APIs developed in the
+context of [Project Fugu](/capabilities/) 🐡 such as [File Handling](/articles/file-handling/),
+[File System Access](/articles/file-system-access/),
+[Async Clipboard](https://web.dev/async-clipboard/), and [Web Share](https://web.dev/web-share/)
+have made this flow possible.
+
+With this in mind, I have developed a test application with the tongue-in-cheek name
+[How Fugu is my browser?](https://howfuguismybrowser.dev/) that you can take for a spin to see what
+Project Fugu APIs your browser of choice supports. Since not all features are exposed on all
+platforms—for example, the [Contact Picker API](/articles/contact-picker/) is currently only exposed
+on mobile—it's technically impossible to reach a score of 100% if you test on desktop (and vice
+versa), so regard it as more like a playful competition than absolute science. For each tested
+feature, there's a link to the relevant documentation so you can learn more about the feature. Where
+feature detection is possible, there is also a note on whether the feature is supported by your
+browser or not, and finally
+[page load statistics linked to Chrome Status](https://chromestatus.com/metrics/feature/timeline/popularity)
+that tell you how popular over time a given feature is.
+
+If your browser supports the Web Share API, you can share how Fugu your browser is by clicking the
+**Share** button right next to the Fugu fish and the progress bar. Else, you can download a
+screenshot, and then share it manually on your online social network of choice.
+
+<figure>
+  {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/8z3ZhxPAq5NxfXEss1XS.png", alt="Screenshot of the site https://howfuguismybrowser.dev/.", width="800", height="649" %}
+  <figcaption>
+    Have you ever wondered: "<a href="https://howfuguismybrowser.dev/">How Fugu is my browser?</a>"
+  </figcaption>
+</figure>
+
+## How Fugu is the Web?
+
+Knowing how Fugu your browser is is only half of the equation. Equally interesting is knowing how
+Fugu the web is. To find an answer to this question, I have developed a browser extension with a
+likewise tongue-in-cheek name, aptly called
+[How Fugu is the web?](https://chrome.google.com/webstore/detail/how-fugu-is-the-web/apcghpabklkjjgpfoplnglnjghonjhdl).
+
+When you install this extension from the Chrome Web Store and browse the web, you will notice that
+the Fugu fish counter on some sites displays a badge with the detected Project Fugu APIs. For
+example, if you browse to [Excalidraw](https://excalidraw.com/), the counter jumps to 9, since
+Excalidraw uses nine detectable Project Fugu APIs. These are:
```

Comment:
```suggestion
Install this extension from the Chrome Web Store and browse the web, then notice how
the Fugu fish counter on some sites displays a badge with the detected Project Fugu APIs. For
example, if you browse to [Excalidraw](https://excalidraw.com/), the counter jumps to 9, since
Excalidraw uses nine detectable Project Fugu APIs. These are:
```